config/winston.js

```diff
@@ -0,0 +1,47 @@
+import appRoot from 'app-root-path';
+import winston from 'winston';
+
+const errorLog = `${appRoot}/logs/error.log`;
+const combinedLog = `${appRoot}/logs/combined.log`;
+const optionsFiles = [
+  {
+    level: 'error',
+    filename: errorLog,
+    handleExceptions: true,
+    json: true,
+    maxsize: 5242880,
+    maxFiles: 5,
+    colorize: false,
+  }, {
+    level: 'warn',
+    filename: combinedLog,
+    handleExceptions: true,
+    json: true,
+    maxsize: 5242880,
+    maxFiles: 5,
+    colorize: false,
+  },
+];
+const optionsConsole = {
+  level: 'debug',
+  handleExceptions: true,
+  json: false,
+  colorize: true,
+  format: winston.format.simple(),
+};
+
+const logger = winston.createLogger({
+  level: 'info',
+  format: winston.format.json(),
```

Comment:
I believe we can enhance the format of the displayed log by using the following code:

```suggestion
  format: format.combine(
    format.timestamp(),
    format.align(),
    format.printf(formatParams),
  ),
```